tests/system/Database/Live/GetTest.php

```diff
@@ -95,6 +95,73 @@ public function testGetFieldData()
 
 		$this->assertEquals('id', $jobs[0]->name);
 		$this->assertEquals('name', $jobs[1]->name);
+
+		$type_test = $this->db->table('type_test')
+						 ->get()
+						 ->getFieldData();
+		
+				if ($this->db->DBDriver === 'SQLite3')
```

Comment:
Please align the indent with line 104.

Reply:
I hope it correct now, I had the special chars on now to see tabs and spaces, sorry you have to babysit me like this.